pkg/local/index.go

```diff
@@ -35,6 +36,20 @@ func (l *Index) GetCurrentURI() (*uri.URI, error) {
 	return uri.Parse(l.PathURI)
 }
 
+func WriteActiveOperation(path string, operation string) error {
```

Comment:
Asking again, how is WriteActiveOperation different from WriteIndex?
Just add the operation parameter to the WriteIndex Function

Reply:
It allows me override just this specific parameter

Reply:
But you are writing the entire file again anyway - the same argument can be done when we update the AtHead value

Reply:
In order to use the WriteIndex function, I have to write these block code every time I want to override the ActiveOperation field:
```
	idx, err = local.ReadIndex(localPath)
	if err != nil {
		if errors.Is(err, fs.ErrNotExist) {
			DieFmt("Failed to read index file from path %s", localPath)
		}
		DieErr(err)
	}
	pathURI, err := idx.GetCurrentURI()
	if err != nil {
		DieFmt("Failed to get PathURI index file.")
	}
	_, err = local.WriteIndex(idx.LocalPath(), pathURI, idx.AtHead, "")
	if err != nil {
		DieErr(err)
	}
```

Reply:
You are reading the index anyway at the beginning of each operation - you can use it

Reply:
But it content changed during the run